### PR TITLE
Chore/app fixes

### LINKED ISSUE #137: Token expired error
Token expires after a while (1 hour ?) and is not refreshed correctly

### PR DESCRIPTION
**Pull request by mheggelund**
closes #137

### CODE REVIEW

File: src/components/Table.tsx [hunk 1]
```diff
@@ -10,14 +10,14 @@ import { useAccessToken } from '../hooks/useAccessToken';
 import { AreaCoordinates } from './AreaCoordinates/AreaCoordinates';
 import * as Styled from './Table.styled';
 
-export const Table = () => {
+export const Table = ({ refetch }: { refetch: number }) => {
```
Comment (Sinrefvol) [inline src/components/Table.tsx:13]: What does it do? How dies it work with UseQuery?

File: src/pages/Browse/Browse.tsx [hunk 2]
```diff
@@ -40,6 +40,7 @@ export const Browse = () => {
   const [uploadId, setUploadId] = useState<string>('');
   const [isAddModelDialog, setAddModelDialog] = useState<boolean>(false);
   const [uploadStatus, setUploadStatus] = useState<string>();
+  const [refetch, setRefetch] = useState<number>(0);
```
Comment (Sinrefvol) [inline src/pages/Browse/Browse.tsx:43]: Will need to explain me this one

File: src/components/Table.tsx (hunk 1 above)
Comment (mheggelund) [reply src/components/Table.tsx:13]: UseQuery takes (here called "refetch") as a parameter it depends on, and when the parameter change useQuery notice it changes and refetch

File: src/pages/Browse/Browse.tsx (hunk 2 above)
Comment (mheggelund) [reply src/pages/Browse/Browse.tsx:43]: To trigger a refetch and update the table of models when a new model is created, we update the state which is connected to the query to trigger refetching og new updated data

**Review by Sinrefvol — APPROVED** [review]

**Pull request merged**